### ISSUE TITLE
Recently Found App Icon... 

### ISSUE THREAD

**Issue by Ibuprophen** [ltguillaume/droidshows#61]
Hello Again My Friend @ltGuillaume! 

I remember when you were considering some input regarding a new icon for the app and I was going to submit a few but, I had... lost them in my backup drive and ended up not submitting them at the time. 

Well I had just found the 2 concept images and here they are in a zip file for you if you want them... 

[DroidShows Icon Concepts.zip](https://github.com/ltGuillaume/DroidShows/files/2294428/DroidShows.Icon.Concepts.zip)

If you do like them and would like them tweaked a bit, please let me know. 

Enjoy! 👍

~Ibuprophen

**Comment by ltguillaume** [comment]
Hey, thanks for the feedback. The idea of the new icon was for it to be up to the Android 8+ standards and to fit in nicely with the Material Design guidelines. I think we've managed to do so quite nicely back then, so the current icon will remain as is.

I'm currently a bit demotivated by the save-to-external-sd stuff I can't seem to get working without all the Android support library stuff, otherwise I would have released a new version months ago. I'm more enthusiastic about working a bit more on my other project https://github.com/ltGuillaume/MusicFolderPlayer which is almost done (volume and progress slider, that's basically all that's left, while all library navigation stuff, filtering, playlist management, after-playlist-management, WhatsApp sharing etc. etc. etc. is all up and running).

**Comment by Ibuprophen** [comment]
The zip images are a 1/2 reduced in size but, it's yours for the taking. Just let me know if you want the original sizes. 

I've had my hands full as well with released and some unreleased Android Developments as well. I've even been giving the developers a hand with the Open Shell software that took over the Classic Shell development. It's great to get back to Windows development since I used to hold a few Microsoft certifications and was certified a Microsoft OEM system builder. That was before I had retired. 

I'll have to check out your other development... Hmmm.. LOL! 


~Ibuprophen
